### PR TITLE
Cu 86b11czg7 profile UI

### PR DESCRIPTION
**Pull request by ReinhardtBotha**
There are a few components I created/updated in this PR, including:

- Box component with the border radius of the cards
- Updated ProgressBar with new styling props
- Badge SVG component
- Image SVG component
- Streak checkbox indicator component
- New button variant
- Divider

### CODE REVIEW

**Comment by zeckkimon** [comment]
Task linked: [CU-86b11czg7 Profile UI](https://app.clickup.com/t/86b11czg7)

File: src/components/ui/Box.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const Box = styled(View, {
+  display: "flex",
+  width: "100%",
+  padding: 16,
+  backgroundColor: globalStyles.colors.white,
+  borderRadius: 16,
+  borderStyle: "solid",
+  borderWidth: 1,
+  borderColor: globalStyles.colors.neutral[100],
```
Comment (whamester) [inline src/components/ui/Box.tsx]: can we pass the colors as props so they can be customized?

https://fedemartinm.github.io/fast-styles/docs/basics/style-props

**Review by whamester — APPROVED** [review]

File: src/components/ui/Box.tsx (hunk 1 above)
Comment (ReinhardtBotha) [reply src/components/ui/Box.tsx]: Ok sure, added here with default values: https://github.com/LangaraBeetles/weaup-react-native-app/pull/42/commits/896746e2c12e512d4351dadd13d4dcb5651f870d

**Review by zeckkimon — APPROVED** [review]
looks great!

**Review by Victorportus — APPROVED** [review]

**Pull request merged**